samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/CSVLayers.razor

```diff
@@ -14,7 +14,7 @@
 <button disabled="@(!_mapRendered)" @onclick="AddLayer">Add new CSV Layer!</button>
 
 
-<SceneView @ref="_view" class="map-view" OnMapRendered="OnMapRendered">
+<SceneView @ref="_view" class="map-view" OnViewRendered="OnMapRendered">
```

Comment:
Do you think it will be good to adjust the other razor pages that use `SceneView` elements to also switch from `OnMapRendered` to `OnViewRendered` for clarity/consistency?  Like the `SceneView` in `ReactiveUtils.razor`.

Reply:
`OnMapRendered` is obsolete, we should update all our samples to `OnViewRendered` (that is irrespective of `MapView` vs. `SceneView`, they're both `Views`).